code/nodes/modelGeneration/funcs.q

```diff
@@ -62,11 +62,11 @@ modelGeneration.modelPrep:{[cfg;mdlDict;tgt]
 // @return     {<} the appropriate function or projection in the case of sklearn
 modelGeneration.mdlFunc:{[lib;fnc;mdl]
   $[lib in key models;
-    get".automl.models.",string[lib],".fitScore";
+    utils.qpyFuncSearch".automl.models.",string[lib],".fitScore";
```

Comment:
Similar to others the python functions aren't going to support dot namespace, so can probably just be a get